core/src/main/scala/org/apache/spark/util/collection/OpenHashSet.scala

```diff
@@ -37,7 +37,6 @@ import com.google.common.hash.Hashing
  * It uses quadratic probing with a power-of-2 hash table size, which is guaranteed
  * to explore all spaces for each key (see http://en.wikipedia.org/wiki/Quadratic_probing).
  */
-private[spark]
```

Comment:
Ah, gotcha.  You can use the new `@Private` annotation on this class to indicate that users still shouldn't use it directly.